polar-c-api/src/lib.rs

```diff
@@ -72,9 +74,27 @@ pub extern "C" fn polar_get_error() -> *const c_char {
     })
 }
 
+lazy_static! {
```

Comment:
Should this be thread-local? Otherwise you'll potentially be getting messages for different threads or instances of oso?

Reply:
Yeah I think so, updated.

They're actually both wrong though because you'd still get errors or messages from the wrong instance if you were interleaving usage of 2 or more polar instances on the same thread. We probably want to make both of those fields on the polar object itself. That doesn't mesh with the way we've organized the ffi code though so is actually a bigger change and orthogonal to this. I think I'll keep it like this for now and do that as part of a follow up.

Reply:
I ended up doing this for messages after all because it was easier for wasm.